subsys/bluetooth/mesh/shell/shell_onoff_cli.c

```diff
@@ -0,0 +1,107 @@
+/*
+ * Copyright (c) 2022 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+ */
+
+#include <stdlib.h>
+#include <bluetooth/mesh/models.h>
+#include <shell/shell.h>
+
+#include "mesh/net.h"
+#include "mesh/access.h"
+#include "shell_utils.h"
+
+static struct bt_mesh_model *mod;
+
+static int shell_onoff_get(const struct shell *shell, size_t argc, char *argv[])
+{
+	if (!mod) {
+		if (!shell_model_first_get(BT_MESH_MODEL_ID_GEN_ONOFF_CLI, &mod)) {
+			return -ENODEV;
+		}
+	}
+
+	struct bt_mesh_onoff_cli *cli = mod->user_data;
+	struct bt_mesh_onoff_status rsp;
+
+	int err = bt_mesh_onoff_cli_get(cli, NULL, &rsp);
+
+	if (!err) {
+		shell_print(shell, "Present val: %d, target val: %d, rem time: %d",
+			    rsp.present_on_off, rsp.target_on_off, rsp.remaining_time);
+	}
+
+	return err;
+}
+
+static int onoff_set(const struct shell *shell, size_t argc, char *argv[], bool acked)
+{
+	bool on_off = (bool)strtol(argv[1], NULL, 0);
```

Comment:
Minor: shell.c does a custom thing for booleans, so you can pass `on` and `off` too, seems like that would be a neat util to put in the common model shell file.